ui-components/urldialog.md

```diff
@@ -7,7 +7,7 @@ keywords: dialog urldialog api
 ---
 ## Overview
 
-A URL dialog is a special TinyMCE UI component which loads an external web page inside a dialog (sometimes referred to as modals). This differs to regular dialogs which use supported components to render an interactive dialog inside the application. URL dialogs are useful for very complex use cases, where the supported dialog components aren't able to be used, for example in a custom file manager.
+A URL dialog is a special TinyMCE UI component which loads an external web page inside a dialog (sometimes referred to as `modals`). This differs from the regular dialogs that use supported components to render an interactive dialog inside the application. URL dialogs are useful for very complex use cases, where the supported dialog components cannot be used — for example, dialogs in a custom file manager.
```

Comment:
@shikhanansi this looks to have changed the meaning? As it's not dialogs *in* a custom file manager, but instead the custom file manager is a separate page that's loaded inside a TinyMCE dialog.

Reply:
Fixed